docs/modules/ROOT/pages/network/network-policies.adoc

```diff
@@ -0,0 +1,37 @@
+= Network Policies
+
+As a Kubernetes Network Plugin plugin Cilium integrates transparently with standard Kubernetes https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.32/#networkpolicy-v1-networking-k8s-io[`NetworkPolicies`].
```

Comment:
This does not fit here. Not all of our clusters use Cilium. Just make a note that there is an extended version here, link to the Cilium page, and move the extended description to the Cilium page.